cypress/component/admin/SortIcon.cy.jsx

```diff
@@ -0,0 +1,44 @@
+import SortIcon from "../../../src/components/admin/dashboards/SortIcon.jsx";
```

Comment:
refer to the other tests to see how they r implemented, there's quite a few things wrong here.

Reply:
used html ID instead of data-cy tags, because some were data-cy tags already present in the SortIcon component file. And when using that tag via .get() Cypress said it couldn't find the tag

Reply:
everything should be via data-cy tags, if there r existing ones, use those, otherwise create new ones

Reply:
I tried to create another tag data-cy tag, but it says that you can only have one per element. And when I used the existsing one it gives the error of Timed out retrying after 4000ms: Expected to find element: [data-cy="name-sort-up"], but never found it.